apps/web/src/modules/auction/components/CurrentAuction/PlaceBid.tsx

```diff
@@ -182,16 +183,38 @@ export const PlaceBid = ({
               </Box>
             </Box>
           </form>
-
-          <ContractButton
-            className={auctionActionButtonVariants['bid']}
-            handleClick={handleCreateBid}
-            disabled={address && isValidChain ? !isValidBid : false}
-            mt={{ '@initial': 'x2', '@768': 'x0' }}
-          >
-            Place bid
-          </ContractButton>
-        </Fragment>
+          <Flex mt="x2">
+            <ContractButton
+              className={auctionActionButtonVariants['bid']}
+              handleClick={handleCreateBid}
+              disabled={address && isValidChain ? !isValidBid : false}
+              mt={{ '@initial': 'x2', '@768': 'x0' }}
+            >
+              Place bid
+            </ContractButton>
+            {chain.id !== 1 ? (
+              <Button
+                ml="x2"
+                mt={{ '@initial': 'x2', '@768': 'x0' }}
+                onClick={async () => {
+                  const baseUrl = `https://nouns.build/dao/${chain.name.toLowerCase()}/${addresses.token}`
```

Comment:
this needs to change for testnet / mainnet. current this is only working with mainnet. you can user `import { PUBLIC_IS_TESTNET } from 'src/constants/defaultChains'` for that.